Source/Service/Models/NotificationCredential.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-public class CotterNotificationCredential: Codable {
+public struct CotterNotificationCredential: Codable {
```

Comment:
I made this change so that it matches all the other Model types, is fine right?

Reply:
yep, that's fine